<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>play_window</class>
 <widget class="QDialog" name="play_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar{
background:white;
  width: 10px;
}
QScrollBar::handle{
background:rgb(180, 180, 180, 150);
}
QScrollBar::handle:hover{
background:rgb(150, 150, 150, 180);
}
QScrollBar::add-page{
background:transparent;
}
QScrollBar::sub-page{
background:transparent; 
}
QScrollBar::sub-line{
background:transparent;
}
QScrollBar::add-line{
background:transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="min_btn">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#min_btn{
border-image: url(:/img/min.png);
background-color: transparent;
}
QPushButton#min_btn::hover{
background-image: url(:/img/min_hover.png);
background-color: transparent;
}

QPushButton#min_btn::pressed{
background-image: url(:/img/min_pressed.png);
background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-image: url(:/img/close.png);
background-color: transparent;
}
QPushButton::hover{
background-image: url(:/img/close_hover.png);
background-color: transparent;
}

QPushButton::pressed{
background-image: url(:/img/close_pressed.png);
background-color: transparent;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QWidget" name="board" native="true">
       <property name="minimumSize">
        <size>
         <width>739</width>
         <height>735</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>739</width>
         <height>735</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#board{
background-image: url(:/img/2063.bmp);
}
</string>
       </property>
       <widget class="QWidget" name="player_top" native="true">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>101</y>
          <width>191</width>
          <height>262</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>191</width>
          <height>262</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#player_top{background-image: url(:/img/2083.bmp);
}</string>
        </property>
        <widget class="QLabel" name="my_chess">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>190</y>
           <width>32</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="pic_1">
         <property name="geometry">
          <rect>
           <x>16</x>
           <y>0</y>
           <width>160</width>
           <height>160</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="name_1">
         <property name="geometry">
          <rect>
           <x>67</x>
           <y>170</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLCDNumber" name="time_left_1">
         <property name="geometry">
          <rect>
           <x>65</x>
           <y>200</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="player_bottom" native="true">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>377</y>
          <width>191</width>
          <height>262</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>191</width>
          <height>262</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#player_bottom{background-image: url(:/img/2124.bmp);
}</string>
        </property>
        <widget class="QLabel" name="his_chess">
         <property name="geometry">
          <rect>
           <x>14</x>
           <y>40</y>
           <width>32</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="pic_2">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>99</y>
           <width>160</width>
           <height>160</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="name_2">
         <property name="geometry">
          <rect>
           <x>67</x>
           <y>15</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLCDNumber" name="time_left_2">
         <property name="geometry">
          <rect>
           <x>64</x>
           <y>46</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Outline</enum>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="play_btn">
        <property name="geometry">
         <rect>
          <x>442</x>
          <y>677</y>
          <width>56</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:transparent;
border-image:url(:/img/2072.bmp) 0 112 0 0;
}
QPushButton:hover{
background-color:transparent;
border-image:url(:/img/2072.bmp) 0 56 0 56;
}
QPushButton:pressed{
background-color:transparent;
border-image:url(:/img/2072.bmp) 0 0 0 112;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="regret_btn">
        <property name="geometry">
         <rect>
          <x>211</x>
          <y>658</y>
          <width>43</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:transparent;
border-image:url(:/img/2068.bmp) 0 86 0 0;
}
QPushButton:hover{
background-color:transparent;
border-image:url(:/img/2068.bmp) 0 43 0 43;
}
QPushButton:pressed{
background-color:transparent;
border-image:url(:/img/2068.bmp) 0 0 0 86;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="lose_btn">
        <property name="geometry">
         <rect>
          <x>516</x>
          <y>672</y>
          <width>46</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:transparent;
border-image:url(:/img/2070.bmp) 0 92 0 0;
}
QPushButton:hover{
background-color:transparent;
border-image:url(:/img/2070.bmp) 0 46 0 46;
}
QPushButton:pressed{
background-color:transparent;
border-image:url(:/img/2070.bmp) 0 0 0 92;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="ani_label">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>150</y>
          <width>450</width>
          <height>450</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
       <widget class="QPushButton" name="draw_btn">
        <property name="geometry">
         <rect>
          <x>372</x>
          <y>673</y>
          <width>45</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:transparent;
border-image:url(:/img/2067.bmp) 0 90 0 0;
}
QPushButton:hover{
background-color:transparent;
border-image:url(:/img/2067.bmp) 0 45 0 45;
}
QPushButton:pressed{
background-color:transparent;
border-image:url(:/img/2067.bmp) 0 0 0 90;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="win_label">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>150</y>
          <width>331</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:red</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="right_widget" native="true">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="bad_apple">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0px;background-color:rgba(255,255,255,180);</string>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::FixedColumnWidth</enum>
          </property>
          <property name="lineWrapColumnOrWidth">
           <number>80</number>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arial'; font-size:4pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="chat_browser">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>270</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="chat_widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#chat_widget{
border-image: url(:/img/myexpression.bmp);
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="send_btn">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton#send_btn{
	border-image: url(:/img/send.png)0 46 0 0;
}
QPushButton#send_btn::hover{
	border-image: url(:/img/send.png)0 23 0 23;
}
QPushButton#send_btn::pressed{
	border-image: url(:/img/send.png)0 0 0 46;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../img/img.qrc"/>
 </resources>
 <connections/>
</ui>
